internal/router/middlewares/ratelim_test.go

```diff
@@ -28,6 +29,9 @@ func TestLimit1IP(t *testing.T) {
 
 		{name: "success", callRPS: 100, limitRPS: 500, forwardedFor: false},
 		{name: "block-me", callRPS: 1000, limitRPS: 500, forwardedFor: false},
+
+		{name: "allow-me", callRPS: 1000, limitRPS: 500, forwardedFor: false, allow: true},
```

Comment:
adds two more test cases where rps is greater than limit but it never gets 429

Reply:
looks good